docs/source/guides/contributing.md

```diff
@@ -108,6 +108,12 @@ fn pub tests(t: mut Tests) {
 When adding a new test module, follow this structure then add it to
 `libstd/test/main.inko`, following the same style as the existing tests.
 
+To run the stdlib tests:
+
+* first build Inko `cargo build release`
+* enter the libstd directory `cd libstd`
+* Run the built inko tests `../target/release/inko test`
```

Comment:
The following steps make this a bit easier:

> 1. Enter the standard library directory using `cd libstd`
> 2. Run the tests using `cargo run -p inko --release -- test`

This way any Rust related changes are picked up right away.